docs/code/MlNetCookBook.md

```diff
@@ -827,71 +562,53 @@ In the static pipeline API, we provide a set of `onFit` delegates that allow int
 
 This is how we can extract the learned parameters out of the model that we trained:
 ```csharp
-// Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
-// as a catalog of available operations and as the source of randomness.
-var mlContext = new MLContext();
 
 // Step one: read the data as an IDataView.
-// First, we define the reader: specify the data columns and where to find them in the text file.
-var reader = mlContext.Data.CreateTextReader(ctx => (
-        // The four features of the Iris dataset.
-        SepalLength: ctx.LoadFloat(0),
-        SepalWidth: ctx.LoadFloat(1),
-        PetalLength: ctx.LoadFloat(2),
-        PetalWidth: ctx.LoadFloat(3),
-        // Label: kind of iris.
-        Label: ctx.LoadText(4)
-    ),
+//  Retrieve the training data.
+var trainData = mlContext.Data.ReadFromTextFile<IrisInput>(irisDataPath,
     // Default separator is tab, but the dataset has comma.
-    separatorChar: ',');
-
-// Retrieve the training data.
-var trainData = reader.Read(dataPath);
+    separatorChar: ','
+);
 
-// This is the predictor ('weights collection') that we will train.
-MulticlassLogisticRegressionPredictor predictor = null;
-// And these are the normalizer scales that we will learn.
-ImmutableArray<float> normScales;
 // Build the training pipeline.
-var learningPipeline = reader.MakeNewEstimator()
-    .Append(r => (
-        r.Label,
-        // Concatenate all the features together into one column 'Features'.
-        Features: r.SepalLength.ConcatWith(r.SepalWidth, r.PetalLength, r.PetalWidth)))
-    .Append(r => (
-        r.Label,
-        // Normalize (rescale) the features to be between -1 and 1. 
-        Features: r.Features.Normalize(
-            // When the normalizer is trained, the below delegate is going to be called.
-            // We use it to memorize the scales.
-            onFit: (scales, offsets) => normScales = scales)))
-    // Cache data used in memory because the subsequently trainer needs to access the data multiple times.
-    .AppendCacheCheckpoint()
-    .Append(r => (
-        r.Label,
-        // Train the multi-class SDCA model to predict the label using features.
-        // Note that the label is a text, so it needs to be converted to key using 'ToKey' estimator.
-        Predictions: mlContext.MulticlassClassification.Trainers.Sdca(r.Label.ToKey(), r.Features,
-            // When the model is trained, the below delegate is going to be called.
-            // We use that to memorize the predictor object.
-            onFit: p => predictor = p)));
-
-// Train the model. During this call our 'onFit' delegate will be invoked,
-// and our 'predictor' will be set.
-var model = learningPipeline.Fit(trainData);
+var dynamicPipeline =
```

Comment:
>dynamicPipeline [](start = 4, length = 15)

let's drop 'dynamic' prefix, and use 'pipeline' or 'learningPipeline' throughout the file.